currency-rates/internal/handlers/rate_test.go

```diff
@@ -9,9 +9,11 @@ import (
 	"strconv"
 	"testing"
 
-	"github.com/RinaDish/currency-rates/internal/handlers"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
+
+	"github.com/RinaDish/currency-rates/internal/handlers"
+	"github.com/RinaDish/currency-rates/tools"
 )
 
 type successRateClient struct{}
```

Comment:
So this is why better to use interfaces that can be moked. So you don't need to make separate client for every scenario

Reply:
Mocking is not a subject of the current task, let's think about this moment later)

Reply:
But the subject of current task was SOLID and interfaces)